mindmeld/models/helpers.py

```diff
@@ -100,6 +101,50 @@ def create_annotator(config):
         raise KeyError(msg.format(config["annotator_class"]))
 
 
+def create_augmentor(config, language, resource_loader):
```

Comment:
I think this function should be in the `mindmeld/augmentation.py` in a factory class that returns augmenter objects.